src/SharedProvider.tsx

```diff
@@ -7,6 +7,8 @@ const SharedProvider = ({ children }: { children: ComponentChildren }) => {
   const { api } = useNodeInfoState().state;
   const { signer, address } = useGlobalState().walletAccount || {};
 
+  console.log('Provider', api, signer, address);
```

Comment:
nit: Do we need this console.log?

Reply:
No, I was not aware that it was added in this PR. Removed.